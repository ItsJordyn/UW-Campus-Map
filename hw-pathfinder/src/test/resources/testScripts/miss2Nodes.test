# Find a path with Dijkstra's algorithm when
# one node is missing

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A C 1.0
AddEdge simpleGraph C B 1.0

FindPath simpleGraph A D
FindPath simpleGraph B D
FindPath simpleGraph D E