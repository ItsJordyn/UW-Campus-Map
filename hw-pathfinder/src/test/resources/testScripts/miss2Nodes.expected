# Find a path with Dijkstra's algorithm when
# one node is missing

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from A to C in simpleGraph
added edge 1.000 from C to B in simpleGraph

unknown: D
unknown: D
unknown: D
unknown: E